src/app/pcasts/dao/users_dao.py

```diff
@@ -163,3 +163,21 @@ def search_facebook_users(facebook_ids, user_id, offset, max_search \
   for u in users:
     u.is_following = is_following_user(user_id, u.id)
   return users
+
+def add_ignored_friend(to_ignore_id, user_id):
+  ignored_ids = IgnoredUsers.query.filter(IgnoredUsers.user_id == user_id).first()
+  if ignored_ids is None:
+    ignored_ids = IgnoredUsers(user_id=user_id, ignored_fb_ids=to_ignore_id)
+  else:
+    ignored_ids.ignored_fb_ids = ignored_ids.ignored_fb_ids + \
+        ",{}".format(to_ignore_id)
+  db_utils.commit_model(ignored_ids)
+  return True
```

Comment:
i don't see the point of returning True since we never return False, if we're trying to see if an exception was thrown there's already a `success` parameter in the default response of an `AppDevController` that has that info

Reply:
This is correct and I'll fix it. I initially didn't want to send an empty response but i guess that is fine